src/DrakeVisualizer.jl

```diff
@@ -115,7 +115,15 @@ function new_window()
     proc
 end
 
-any_open_windows() = success(spawn(`pgrep $drake_visualizer_executable_name`))
+function any_open_windows()
+    if is_apple()
```

Comment:
You can put `@static` before the `if` to have this magically happen at compile-time